core/object/object.h

```diff
@@ -64,6 +64,7 @@ enum PropertyHint {
 	PROPERTY_HINT_EXP_EASING, /// exponential easing function (Math::ease) use "attenuation" hint string to revert (flip h), "full" to also include in/out. (ie: "attenuation,inout")
 	PROPERTY_HINT_LENGTH, ///< hint_text= "length" (as integer)
 	PROPERTY_HINT_KEY_ACCEL, ///< hint_text= "length" (as integer)
+	PROPERTY_HINT_ANGLE, ///< angle in radians, hints and editing in degrees
```

Comment:
Should this also be added to `modules/gdnative/include/nativescript/godot_nativescript.h`?

Reply:
nativescript is going to die in a couple of days so probably not.